src/screens/ledger.tsx

```diff
@@ -27,7 +27,9 @@ const screenOptionsBiometry: ScreenOptionType = {title, headerBackHidden: true};
 export const LedgerScreen = () => {
```

Comment:
Revert this file pls. it is not related to the main task

Reply:
done